WasatchNET/EEPROM.cs

```diff
@@ -1532,6 +1532,94 @@ public bool read()
                 return true;
             }
 
+            else if (spectrometer is HOCTSpectrometer)

```

Comment:
100% agreed. It's very messy right now. Might be cleaner to have something like getPages(specType spec) but then we'd need to reverse engineer the byte loads. Or have something like a "hasEEPROM" parameter rather than splitting it by type